src/protocol/protocol.ts

```diff
@@ -423,9 +423,9 @@ export namespace CommonDataTypes {
     type: 'function';
   };
 
-  export type RegExpRemoteValue = RemoteReference & RegExpLocalValue;
+  export type RegExpRemoteValue = RegExpLocalValue & RemoteReference;
```

Comment:
I don't know how useful this sort rule is, looking at the code now it would be harder to check what extends what imo.

Reply:
I was on the fence about this one. Disabled it for unions, but for intersections it seemed OK. Let's disable it fully then.